akka-docs/rst/scala/remoting-artery.rst

```diff
@@ -0,0 +1,465 @@
+if they possess an :class:`ActorRef` pointing to those system. This means that every system that is remoting enabled
+acts as a "server" to which arbitrary systems on the same network can connect to.
+
+What is new in Artery
+---------------------
+
+Artery is a reimplementation of the old remoting module aimed at improving performance and stability. It is mostly
+backwards compatible with the old implementation and it is a drop-in replacement in many cases. Main features
+of Artery compared to the previous implementation:
+
+* Based on `Aeron <https://github.com/real-logic/Aeron>`_ instead of TCP
+* Focused on high-throughput, low-latency communication
+* Mostly allocation-free operation
+* Support for a separate "lane" for large messages to avoid interference with smaller messages
+* Compression of actor paths on the wire to reduce overhead for smaller messages
+* Support for faster serialization/deserialization using ByteBuffers directly
+* Serviceability through the built-in Flight-Recorder
+
+The main incompatible change from the previous implementation that the protocol field of the string representation of an
+:class:`ActorRef` is always `akka` instead of the previously used `akka.tcp` or `akka.ssl.tcp`.
+
+Preparing your ActorSystem for Remoting
+---------------------------------------
+
+The Akka remoting is a separate jar file. Make sure that you have the following dependency in your project::
+
+  "com.typesafe.akka" %% "akka-remote" % "@version@" @crossString@
+
+To enable remote capabilities in your Akka project you should, at a minimum, add the following changes
+to your ``application.conf`` file::
+
+  akka {
+    actor {
+      provider = remote
+    }
+    remote {
+      artery {
+        enabled = on
+        canonical.hostname = "127.0.0.1"
+        canonical.port = 2552
+      }
+    }
+  }
+
+As you can see in the example above there are four things you need to add to get started:
+
+* Change provider from ``local`` to ``remote``
+* Enable Artery to use it as the remoting implementation
+* Add host name - the machine you want to run the actor system on; this host
+  name is exactly what is passed to remote systems in order to identify this
+  system and consequently used for connecting back to this system if need be,
+  hence set it to a reachable IP address or resolvable name in case you want to
+  communicate across the network.
+* Add port number - the port the actor system should listen on, set to 0 to have it chosen automatically
+
+.. note::
+
+  The port number needs to be unique for each actor system on the same machine even if the actor
+  systems have different names. This is because each actor system has its own networking subsystem
+  listening for connections and handling messages as not to interfere with other actor systems.
+
+The example above only illustrates the bare minimum of properties you have to add to enable remoting.
+All settings are described in :ref:`remote-configuration-scala`.
```

Comment:
This links to the main configuration page which includes the configuration. That is the correct I think?